boards/teensy31/board.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2017 INRIA
```

Comment:
Minor (e.g non blocking) but it should be `Inria`. `INRIA` is the old school name ;)